myfile.txt

```diff
@@ -0,0 +1,24 @@
+---------------------------------
+News
+Zookeepers have made the heartbreaking decision to put down all the large animals at a zoo in Ukraine after its enclosures were destroyed by shelling.
+Kharkiv, 2022-10-11 23:11
+---------------------------------
+
+---------------------------------
+Private Ad
+I want to rent an apartments
+Actual until: 2022-11-01, 21 days left
+---------------------------------
+
+---------------------------------
+Weather forecast

```

Comment:
Don't see corresponding abstract class or certain class for this publication:)

Reply:
It is same as news, but another name. I'll delete possibility to insert random name by user